draft-ietf-httpbis-digest-headers.md

```diff
@@ -1079,7 +1079,7 @@ registry. The allowed values for the "Status" fields are described below.
 
 ## Deprecate "MD5" Digest Algorithm {#iana-MD5}
 
-This memo updates the "MD5" digest algorithm in the [HTTP Digest Algorithm
+This memo updates the "MD5" digest-algorithm in the [HTTP Digest Algorithm
```

Comment:
@LPardue here and below, we could replace `digest-algorithm` with `entry`

Reply:
Based on your other PR, lowercase again?

Reply:
will be lowercased by the other PR but not in this one.